src/web/mjs/engine/Connector.mjs

```diff
@@ -444,9 +444,9 @@ export default class Connector {
                         .then( () => this.fetchDOM( request, selector, retries - 1 ) );
                 }
                 if( response.status === 200 ) {
-                    return response.text()
+                    return response.arrayBuffer()
                         .then( data => {
-                            let dom = this.createDOM( data );
+                            let dom = this.createDOM( new TextDecoder(encoding).decode(data) );
```

Comment:
Most methods do not provide an encoding, use default fallback to ensure all other connectors will still work
`new TextEncoding(encoding || 'utf8').decode(...)`

Reply:
TextEncoding with undefined seems to do just that
but il put it in anyways

Reply:
Didn't know that TextEncoder falls back to UTF8 as default